src/api/data-migration/version-4-to-version-5/index.ts

```diff
@@ -0,0 +1,30 @@
+import connectToDatabase from '../connect-to-database';
+import updateApplications from './update-applications';
+
+/**
+ * dataMigration
+ * Update all applications from the V4 data model/structure,
+ * to the V5 data model/structure.
+ * @returns {Function} process.exit()
+ */
+const dataMigration = async () => {
+  try {
+    console.info('🚀 Beginning data migration (v4 to v5)');
+
+    const connection = await connectToDatabase();
+
+    await updateApplications(connection);
+
+    console.info('✅ Applications successfully updated');
```

Comment:
Can we add number of rows migrated, will `updateApplications` return an array of rows migrated?